aspnetcore/fundamentals/configuration/index/includes/index3-5.md

```diff
@@ -273,7 +273,7 @@ This topic only pertains to *app configuration*. Other aspects of running and ho
 
 Environment variables set in `launchSettings.json` override those set in the system environment.
 
-For more information on migrating app configuration from earlier versions of ASP.NET, see <xref:migration/proper-to-2x/index#store-configurations>.
+For more information on migrating app configuration from earlier versions of ASP.NET, see <xref:migration/fx-to-core/index#store-configurations>.
```

Comment:
```suggestion
For more information on migrating app configuration from earlier versions of ASP.NET, see <xref:migration/fx-to-core/index>.
```

This store-configurations anchor does not seem to exist on the fx-to-core/index topic. The link will still work, but that section does not exist.